qiskit/providers/models/backendconfiguration.py

```diff
@@ -570,16 +584,78 @@ def acquire(self, qubit: int) -> AcquireChannel:
         controlling multiqubit interactions. This channel is derived from other channels.
 
+        Args:
+            qubits: Tuple or list of qubits of the form `(control_qubit, target_qubit)`.
+            channel: Deprecated.
+
+        Raises:
+            BackendConfigurationError: If the ``qubits`` is not a part of the system.
+
         Returns:
-            Qubit control channel.
+            List of control channels.
         """
         # TODO: Determine this from the hamiltonian.
-        return ControlChannel(channel)
+        if channel is not None:
+            warnings.warn('control(channel: int) is deprecated. Use '
+                          'control(qubits: List or Tuple [control_qubit, target_qubit]) instead. '
+                          'The ControlChannel information is retrieved from the backend. ',
+                          DeprecationWarning)
+            qubits = [channel]
+        try:
+            if isinstance(qubits, list):
+                qubits = tuple(qubits)
+            return self._control_channels[qubits]
+        except KeyError:
+            raise BackendConfigurationError("Couldn't find the ControlChannel operating on qubits "
+                                            "{} on {}-qubit system. The ControlChannel information"
+                                            " is retrieved from the "
+                                            " backend.".format(qubits, self.n_qubits))
+
+    def get_channel_qubits(self, channel: Channel) -> List[int]:
+        """
+        Return a list of indices for qubits which are operated on directly by the given ``channel``.
+
+        Raises:
+            BackendConfigurationError: If ``channel`` is not a found.
+
+        Returns:
+            List of qubits operated on my the given ``channel``.
+        """
+        try:
+            return self._channel_qubit_map[channel]
+        except KeyError:
+            raise BackendConfigurationError("Couldn't find the Channel - {}".format(channel))
+
+    def get_qubit_channels(self, qubit: Union[int, Iterable[int]]) -> List[Channel]:
+        r"""Return a list of channels which operate on the given ``qubit``.
+
+        Raises:
+            BackendConfigurationError: If ``qubit`` is not a found.
+
+        Returns:
+            List of ``Channel``\s operated on my the given ``qubit``.
+        """
+        channels = list()
+        try:
+            if isinstance(qubit, int):
+                for key in self._qubit_channel_map.keys():
+                    if qubit in key:
+                        channels.extend(self._qubit_channel_map[key])
+                qubit = [qubit]
+            if isinstance(qubit, list):
+                qubit = tuple(qubit)
+                channels.extend(self._qubit_channel_map[qubit])
+            elif isinstance(qubit, tuple):
+                channels.extend(self._qubit_channel_map[qubit])
+            return channels
+        except KeyError:
```

Comment:
What if I have this:
```
_qubit_channel_map = {(0, 1): channel_list}
```
and I run:
```
config.get_qubit_channels(0)
```
I think I'll end up with a `KeyError`.

Reply:
The backends probably fill out the channel config for all single qubits, so to test this, you probably have to overwrite the private dictionary `_qubit_channel_map`

Reply:
`config.get_qubit_channels(0)` won't end up with a KeyError. I've added a test for the same in [L78-80](https://github.com/Qiskit/qiskit-terra/pull/4097/files#diff-591b8b8a4236166fab5fa5df97f828c1R78-R80). 

There was a bug -- repeated Channels were returned for `config.get_qubit_channels(0)`. The fixes to this are added in [641a0aa](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/641a0aa9e09f667865a6b5a8072153a238e36f3c). This wasn't detected in the tests because I explicitly converted the returned lists into a set (in tests). I've corrected this in the test in the same (above mentioned) commit.